admin_guide/out_of_resource_handling.adoc

```diff
@@ -47,10 +48,13 @@ description column based on the node summary API.
 |`memory.available` = `node.status.capacity[memory]` - `node.stats.memory.workingSet`
 |===
```

Comment:
Where are these signals actually given out? Is this in the logs? Which files exactly?